include/votca/tools/calculator.h

```diff
@@ -58,7 +58,7 @@ class Calculator {
    *
    * @param options Property object passed by the application to a calculator
    */
-  virtual void Initialize(Property &options) = 0;
+  virtual void Initialize(Property &user_options) = 0;
```

Comment:
okay thi is annoying but should the property not by const?

Reply:
sure, let's do that

Reply:
this will require changing all the options parsing in xtp, because all calculators will have to changed.

Reply:
done